### PR TITLE
Integration Basel-Land

### PR DESCRIPTION
**Pull request by danji90**
Adding new profile Basel-Land

### CODE REVIEW

File: lib/data/sql/V1.2__import_bl.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+COPY bl_standorttypen
+FROM '/data/profiles/bl/standorttypen.csv'
+DELIMITER ';' CSV HEADER;
+
+CREATE TABLE bl_gesellschaftsgruppen (STO_NR TEXT, GESGR_CAT TEXT, GESGR_DEU TEXT, WALDBILD TEXT, STANDORT TEXT, NUTZUNG_PFLEGE TEXT, VEGETATIONSSTUFE TEXT, FLAECHENANTEIL_BL TEXT, FLAECHENANTEIL_BS TEXT, FLAECHENANTEIL_BLBS TEXT, FLAECHE_BLBS_PROZENT TEXT);
+COPY bl_gesellschaftsgruppen
+FROM '/data/profiles/bl/gesellschaftsgruppen.csv'
+DELIMITER ';' CSV HEADER;
+
+-- CREATE TABLE lu_artengruppen (STO_Nr TEXT, A TEXT, B TEXT, C TEXT, E TEXT, F TEXT, G TEXT, H TEXT, I TEXT, J TEXT, K TEXT, L TEXT, M TEXT, N TEXT, O TEXT, P TEXT, BEMERKUNG TEXT, GESGR_Nr TEXT);
```
Comment (friedjoff) [inline lib/data/sql/V1.2__import_bl.sql:11]: I guess this can be removed.

File: lib/data/sql/V6__export_types.sql [hunk 2]
```diff
@@ -346,4 +376,4 @@ COPY
           relief,
           silver_fir_areas,
           slope,
-          treetype))) TO '/data/types.json';
+          treetype))) TO PROGRAM $$sed 's/\\\\\"/\\\"/g' > '/data/types.json'$$;
```
Comment (friedjoff) [inline lib/data/sql/V6__export_types.sql:379]: Maybe add a comment why we do this.

File: src/i18n/resources/de/translation.json [hunk 3]
```diff
@@ -477,6 +478,77 @@
...
+        "m": "Basen- und Kalkzeiger",
+        "n": "Arten auf Blockschutt",
+        "o": "Basenzeiger wechselfeucht / wechseltrocken",
+        "p": "Trockenheitszeiger"
+      },
+      "terrain": "Relief",
+      "cantonalForestType": "Kantonaler Standortstyp"
+    }
+  },
+  "bl": {
```
Comment (friedjoff) [inline src/i18n/resources/de/translation.json:503]: Do we provide translations for BL? For example  the `AssociationsTab` component has untranslated strings. But if BL is only used in German we can skip i18n.

**Review by friedjoff — CHANGES_REQUESTED** [review]
Some minor things.

File: src/i18n/resources/de/translation.json (hunk 3 above)
Comment (danji90) [reply src/i18n/resources/de/translation.json:503]: @ElkeErhardt Is BL going to be German only?

Comment (ElkeErhardt) [reply src/i18n/resources/de/translation.json:503]: Yes there will be no French Version, same like Luzern

File: lib/data/sql/V1.2__import_bl.sql (hunk 1 above)
Comment (danji90) [reply lib/data/sql/V1.2__import_bl.sql]: Sure

File: lib/data/sql/V6__export_types.sql (hunk 2 above)
Comment (danji90) [reply lib/data/sql/V6__export_types.sql]: done

File: src/i18n/resources/de/translation.json (hunk 3 above)
Comment (danji90) [reply src/i18n/resources/de/translation.json]: I removed all i18n content for Basel and hard-code the text content

**Pull request merged**